chapters/mitigations-and-defensive-strategies/defense-mechanisms/activities/04-rwslotmachine4/sol/README.md

```diff
@@ -0,0 +1,43 @@
+# Building and Running
+
+## Using Docker CLI directly
+
+1. Build the Docker image:
+
+    ```bash
+    docker build -t rwslotmachine4 -f Dockerfile .
+    ```
+
+1. Run a Docker instance from the Docker image:
+
+    ```bash
+    docker run -d --rm -p 31347:31337 --name rwslotmachine4-cnt -t rwslotmachine4
+    ```
+
+## Using the Makefile
+
```

Comment:
Remove this part, only leave the `Makefile`-based approach. If someone wants to run manually / via the CLi, they can check the `Makefile`.